content/authorization/sign-in-with-livechat/index.mdx

```diff
@@ -66,268 +73,419 @@ The SDK is available in two formats: an NPM module and a CDN-hosted library.
 
-> Native "Sign in with LiveChat" button
+Prepare the login button, which will invoke the authorization flow when a user clicks it.
 
 ```html
-<div class="livechat-login-button"></div>
+<div id="login-button" style="background: url(.../livechat_sign_in.png)"></div>
 ```
 
-> Custom "Sign in with LiveChat" button
+<CodeSample path={'Prepare the button container'}>
 
 ```js
 // javascript
-const instance = accountsSdk.init({ ... });
+const instance = new AccountsSDK({
+  client_id: '<your_app_client_id>',
+  redirect_uri: '<your_app_redirect_uri>'
+});
+
+document.getElementById('login-button').onclick = (e) => {
+  if (e && e.preventDefault) {
+    e.preventDefault();
+  }
+
+  sdk.popup().authorize().then((authorizeData)=>{
+    const transaction = sdk.verify(authorizeData);
+    if (transaction != null) {
+      // authorization success
+      // authorizeData contains `accessToken` or `code`
+      // transaction contains state and optional code_verifier (code + PKCE)
+      console.log("User access token: " + transaction.accessToken)
+      document.getElementById('login-button').style.display = "none"
+    } else {
+      console.log("Redirect state don't matches previous one")
+    }
+  }).catch((e)=>{
+    console.error("Failed to authorize user", e)
+  })
+};
 ```
 
-```html
-<!-- html -->
-<a href="" onclick="instance.openPopup()">Sign in with LiveChat</a>
+</CodeSample>
+
+During the creation of the Sign in LiveChat button, you can use [LiveChat Designs](https://livechat.design/).
+
+<Warning>
+
+[**Don't forget to test the give consent step**](/monetization/app-review-process/#common-problems). Use `prompt: "consent"` to force the app to **ask you for access** to certain resources. It’s necessary for you to test the app as if you were a user who installs the app from Marketplace.
+
+</Warning>
+
+# Accounts SDK
+
+## Classes
+
+- [AccountsSDK](#accountssdk)
+- [Popup](#popup-1)
+- [Redirect](#redirect-1)
+
+## AccountsSDK
+
+The main instance of the SDK used to authorize users in LiveChat Accounts.
+
+<CodeSample path={'Example class creation'}>
+
+```js
+const instance = new AccountsSDK({
+  client_id: '<your_app_client_id>'
+});
 ```
 
-Insert HTML code to prepare the container for the "Sign in with LiveChat" button. Its `class` attribute must have value of `livechat-login-button`. The button will be automatically inserted into the container.
+</CodeSample>
 
-If you prefer to design your own button, you can do that, too. Just bind the [`openPopup()`](#instanceopenpopup) method of `AccountsSDK` instance to the `onclick` attribute of your link or button.
+The constructor accepts an `options` object with the following properties:
 
-## Step 4: Initialize the SDK
+| Property                      | Required | Data type | Description                                                                                                                                                                                                      |
```

Comment:
I added the column for data type in each table with params. Could you pay attention to them when going through this doc?